R/utilities.R

```diff
@@ -242,11 +242,11 @@ get_ssh_session = function(host="gphost01.bcgsc.ca"){
 #' Get regions from genes.
 #'
 #' @param gene_symbol A vector of one or more gene symbols.
-#' @param ensembl_id A vector of one or more Esemble IDs.
+#' @param ensembl_id A vector of one or more Ensemble IDs.
```

Comment:
There is no e at the end of Ensembl

Reply:
I agree, I changed a typo into another typo :)